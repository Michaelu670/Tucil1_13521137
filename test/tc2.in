5 2 3 Q