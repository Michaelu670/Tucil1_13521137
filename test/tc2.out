Kartu : 5 2 3 Q
1 solution found
12 / (3 - (5 / 2))
Execution time = 3 milliseconds
